projects/plugins/jetpack/extensions/blocks/goodreads/goodreads.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Goodreads Block.
+ *
+ * @since 1.0.0 // Replace with the correct version number.
```

Comment:
You can use `$$next-version$$` as a placeholder here, and everywhere where you need to provide a version. It will be automatically replaced during the release process.

```suggestion
 * @since $$next-version$$
```